SMLHelper/Initializer.cs

```diff
@@ -32,6 +35,15 @@ public static void PrePatch()
             TechTypePatcher.cacheManager.LoadCache();
             Logger.Debug("Loading CraftTreeType Cache");
             CraftTreeTypePatcher.cacheManager.LoadCache();
+
+            try
+            {
+                PreInitialize();
```

Comment:
If you're going to create a new method to add prepatch orchestration, move everything into either the `PreInitialize` method or into `PrePatch` directly.